.github/workflows/tutorial_docker.yaml

```diff
@@ -21,8 +21,8 @@ jobs:
       packages: write
       contents: read
     env:
-      GH_IMAGE: ghcr.io/ros-planning/moveit2:main-tutorial-${{ matrix.ROS_DISTRO }}-${{ github.job }}
-      DH_IMAGE: moveit/moveit2:main-tutorial-${{ matrix.ROS_DISTRO }}-${{ github.job }}
+      GH_IMAGE: ghcr.io/ros-planning/moveit2:main-${{ matrix.ROS_DISTRO }}-${{ github.job }}
+      DH_IMAGE: moveit/moveit2:main-${{ matrix.ROS_DISTRO }}-${{ github.job }}
```

Comment:
@MarqRazz here is the real technical change.

ROS_DISTRO expands to `humble`, `rolling`, or `iron`
github.job expands to `tutorial-source`

so a full name will in the form `main-humble-tutorial-source`